PatchesAndAreasApi/PatchesAndAreasApi.csproj

```diff
@@ -35,7 +35,7 @@
     <PackageReference Include="Hackney.Core.Sns" Version="1.52.0" />
     <PackageReference Include="Hackney.Core.Validation" Version="1.56.0" />
     <PackageReference Include="Hackney.Core.Validation.AspNet" Version="1.53.0" />
-    <PackageReference Include="Hackney.Shared.PatchesAndAreas" Version="0.9.0" />
+    <PackageReference Include="Hackney.Shared.PatchesAndAreas" Version="0.13.0" />
```

Comment:
Whoah! That's a very big jump.

Within this PR's description, could link the PRs for the shared package's m.v.: 10, 11, 12, and 13 (for completeness) releases?

So we know what we're signing up for. And so that if anything goes wrong, that it would be easy to track down & investigate what kind of changes happened since m.v. 9 release.

Reply:
It's difficult to tell but I believe this is the diff: https://github.com/LBHackney-IT/patches-and-areas-shared/compare/9937e5ba2bc07906d5e758191f6fa9d9b27a09a7...main

1. https://github.com/LBHackney-IT/patches-and-areas-shared/pull/5 - v10
2. https://github.com/LBHackney-IT/patches-and-areas-shared/pull/6 (reverts the previous) - v11
3. https://github.com/LBHackney-IT/patches-and-areas-shared/pull/7 (my PR with the enum change) - v12
4. https://github.com/LBHackney-IT/patches-and-areas-shared/pull/8 (my PR with the docker-compose change) - v13

Reply:
For small changes like the pipeline fix you can use a branch without the `feature/` prefix.
Shame on me for not noticing it was a feature branch :face_exhaling: 
The branches without `feature/` prefix don't trigger deployment.
Though, I don't suppose much can be done about the 'add feature' and then 'revert feature' :thinking: